camelot/view/action_steps/crud.py

```diff
@@ -91,6 +94,8 @@ def get_delegate_state(self, static_field_attributes):
         elif issubclass(fa['delegate'], delegates.One2ManyDelegate):
             attrs = filter_attributes(fa, ['admin_route', 'column_width', 'columns', 'rows',
                                                 'action_routes', 'list_actions', 'list_action'])
+            attrs['group'] = get_settings_group(attrs['admin_route'])
+            attrs['crud_actions'] = CrudActions(None) # FIXME: admin arg
```

Comment:
good question : moet die admin hier gewoon weg als arg, of hebben we die just wel nodig ...